src/GenFacades/Program.cs

```diff
@@ -314,7 +314,7 @@ private static IEnumerable<string> EnumerateDocIdsToForward(IAssembly contractAs
             // Make note that all type forwards (including nested) implement INamespaceAliasForType, so do
```

Comment:
IIRC, we used to filter to namespace types only and add nested forwarders with special logic to replicate the csc behavior, which included private nested types of the destination. Then we decided to change it so that we only forward to types that are actually defined in the reference assembly. See https://github.com/dotnet/buildtools/pull/275. 

I did not study things, but I suspect we broke something in that move. This comment does seem out of date. I'd remove it.